lib/Doctrine/ORM/Query/Expr.php

```diff
@@ -526,13 +526,13 @@ public function notLike($x, $y)
      * Creates a CONCAT() function expression with the given arguments.
      *
      * @param mixed $x First argument to be used in CONCAT() function.
-     * @param mixed $y Second argument to be used in CONCAT() function.
+     * @param mixed $y,... Other arguments to be used in CONCAT() function.
      *
      * @return Expr\Func
      */
     public function concat($x, $y)
```

Comment:
@giosh94mhz If the class were *actually* marked final, it would not matter. Given that it is not actually marked final, someone out there may have extended it. As part of the public API, the intent for this class doesn't really matter. The class is not final, so removing a parameter of this function is indeed a BC break (will raise warnings if it has been extended)